rules/internal/framework_middleman.bzl

```diff
@@ -188,9 +237,13 @@ def _dep_middleman(ctx):
     cc_providers = []
     avoid_libraries = {}
 
+    _is_bazel_7 = not hasattr(apple_common, "apple_crosstool_transition")
```

Comment:
Mind updating all usage of this to use `utils.bzl` `is_bazel_7`, i landed that recently.